### PR TITLE
Fix infinite loop problem

### PR DESCRIPTION
**Pull request by bobbyiliev**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ Feature
- [x] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🚩 Other

## Description

Fixing the bug noticed during the weekly demo

## Related Tickets & Documents

N/A

## Added to documentation?

- [ ] 📜 readme
- [x] 🙅 no documentation needed

### CODE REVIEW

File: datagen.ts [hunk 1]
```diff
@@ -28,9 +28,9 @@ program
     )
     .addOption(
         new Option(
-            '-n, --number <char>',
+            '-n, --number <number>',
```
Comment (chuck-alt-delete) [inline datagen.ts:31]: could this be `<int>` instead?

**Review by chuck-alt-delete — CHANGES_REQUESTED** [review]

File: datagen.ts (hunk 1 above)
Comment (chuck-alt-delete) [inline datagen.ts]:
```suggestion
            '-n, --number <int>',
```

Comment (bobbyiliev) [reply datagen.ts]: Yep good point!

**Review by chuck-alt-delete — APPROVED** [review]

**Comment by chuck-alt-delete** [comment]
I’m going to go ahead and merge so I can get this running for an experiment

**Pull request merged**